package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "throwcast-angular",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/darenang44/throwcast-angular.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/darenang44/throwcast-angular/issues"
+  },
+  "homepage": "https://github.com/darenang44/throwcast-angular#readme",
+  "dependencies": {
+    "angular": "^1.5.8",
+    "body-parser": "^1.15.2",
```

Comment:
We probably don't need some of these dependencies.

* body-parser - We won't be sending any requests to this server other requests for static files.
* morgan - We don't need to log the requests for static files.
* nodemon - We are only serving static files so we don't need to restart the server.